scripts/build/download-tool.sh

```diff
@@ -78,7 +78,8 @@ if [ -f "$ARGV_OUTPUT" ]; then
   fi
 fi
 
-wget --no-check-certificate "$ARGV_URL" -O "$TEMP_OUTPUT"
+echo "Downloading $ARGV_URL"
+curl --continue-at - --retry 100 --location --output "$TEMP_OUTPUT" "$ARGV_URL"
```

Comment:
This is the only place where `curl` will download "big" files (beyond text files with a few kilobytes), which is the reason I only included those options here. Do you think we should still add those options everywhere?

> But I wonder if a retry of 100 is a little excessive?

A retry of 10 was still failing very often for me. Maybe better to be on the safe side, at least for me? :P